config/samples/storageos-etcd-cluster.yaml

```diff
@@ -51,3 +51,13 @@ spec:
                 values:
                   - storageos-etcd
           topologyKey: kubernetes.io/hostname
+        preferredDuringSchedulingIgnoredDuringExecution:
+        - weight: 1
```

Comment:
I think weight 1 is too low, this should be changed to 100. A weight of 1 is almost inconsequential: https://aws.amazon.com/blogs/developer/aws-cli-v2-is-now-generally-available/

Reply:
> I think weight 1 is too low, this should be changed to 100.

We plan to patch this with kustomize if the user sets a topology key when installing via the plugin

We're only adding it here (and therefore setting it as default), as limitations with the plugin make it difficult to add an entire new entire object via a kustomize patch.